### PR TITLE
feat(overlay): allow to set the overlay style

### LINKED ISSUE #15: [FEAT] Allow to set the Overlay style
In version `0.1.1`, it is hard coded and cannot be changed. We set a specific style depending if the BPMN element is a edge/flow or a shape.

### PR DESCRIPTION
**Pull request by csouchet**
- Add new functions to style an overlay: `create_fill`, `create_font`, `create_stroke` & `create_style`
- Add new parameter on `display` function: `enableDefaultOverlayStyle`. If no style is set on an overlay, and this parameter is set to `TRUE`, the default style will be applied to the overlay.
- Update the documentation
- Update the generated documentation
- Add tests

Closes #15

**NOTES**
The different tracks I used for my reflexion:
- https://cengel.github.io/R-intro/gettingstarted.html#list
- https://stackoverflow.com/questions/55000237/json-array-in-body-parameter-of-httrpost
- https://stackoverflow.com/questions/70883767/passing-json-into-r-function
- https://cran.r-project.org/web/packages/jsonlite/vignettes/json-aaquickstart.html

### CODE REVIEW

File: inst/htmlwidgets/bpmnVisualizationR.js [hunk 1]
```diff
@@ -24,42 +24,22 @@ HTMLWidgets.widget({
...
             renderValue: function(x) {
                 bpmnVisualization.load(x.bpmnContent, { fit: {type: bpmnvisu.FitType.Center, margin: 30} });
-
+                
                 // Add overlays
-                x.overlays && x.overlays.map(overlay => {
+                x.overlays?.map(overlay => {
                     const elementsByIds = bpmnVisualization.bpmnElementsRegistry.getElementsByIds(overlay.elementId);
+                    
+                     console.log(overlay.style);
```
Comment (tbouffard) [inline inst/htmlwidgets/bpmnVisualizationR.js:32]: **issue:**  no-console log
```suggestion
```

File: inst/htmlwidgets/bpmnVisualizationR.js [hunk 2]
```diff
@@ -24,42 +24,22 @@ HTMLWidgets.widget({
         return {
             renderValue: function(x) {
                 bpmnVisualization.load(x.bpmnContent, { fit: {type: bpmnvisu.FitType.Center, margin: 30} });
-
+
```
Comment (tbouffard) [inline inst/htmlwidgets/bpmnVisualizationR.js:27]: **nitpick:** lint
```suggestion

```

File: inst/htmlwidgets/bpmnVisualizationR.js (hunk 1 above)
Comment (csouchet) [reply inst/htmlwidgets/bpmnVisualizationR.js:32]: Good catch :smile:

**Review by tbouffard — APPROVED** [review]
LGTM. Minor suggestions for the documentation.

**Pull request merged**